lib/net_mosq.c

```diff
@@ -34,7 +34,7 @@ and the Eclipse Distribution License is available at
 #include <sys/endian.h>
 #endif
 
-#ifdef __FreeBSD__
+#if defined(__FreeBSD__) || defined(__NetBSD__)
```

Comment:
Maybe but when i look on the patches in Ports and Pkgsrc, is this maybe a smaller problem. For example NetBSD doesn't support long options for install, this should also the case for all BSDs.